qa/L0_custom_ops/test.sh

```diff
@@ -105,6 +106,64 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+## Runtime loading via config
+SERVER_ARGS="--model-repository=tf_custom_ops --strict-model-config=false"
+SERVER_ARGS="--model-repository=tf_custom_ops"
+SERVER_LD_PRELOAD=""
+
+rm -rf tf_custom_ops && \
+    mkdir -p tf_custom_ops && \
+    cp -r /data/inferenceserver/${REPO_VERSION}/qa_custom_ops/tf_custom_ops .
+
+echo "model_operations { op_library_filename: \"tf_custom_ops/libbusyop.so\" }" >> tf_custom_ops/graphdef_busyop/config.pbtxt
```

Comment:
It makes more sense to loop MODEL_TYPE between graphdef and savedmodel. Then copy the respective custom ops files.